hawc/apps/lit/templates/lit/reference_confirm_delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'lit/reference_detail.html' %}
+
+{% block title %}
+  {{ block.super }} | Delete
+{% endblock title %}
+
+{% block breadcrumbs_self %}
+  <li><a href="{{object.get_absolute_url}}">{{object}}</a><span class="divider">/</span></li>
+  <li class="active"> Delete <span class="divider">/</span></li>
+{% endblock breadcrumbs_self %}
+
+{% block content %}
+  {{ block.super }}
+  {% include "hawc/_delete_block.html" with name="reference" notes="This will remove all instances of this reference from this assessment." %}
```

Comment:
Add same as above: `This also removes the reference from any searches/imports which may have included the reference. If data was extracted with this reference and it is associated with bioassay, or epi extractions they will also be removed.`